pages/overview.mdx

```diff
@@ -0,0 +1,47 @@
+import { PortalUSP } from 'components/pop'
+
+## What is the Portal Network?
+
+<br />
+<br />
+<PortalUSP />
+<br />
+
+Ethereum users have two options: 
+
+- dedicate a non-trivial amount of computing resources and bandwidth to running an Ethereum node
+- trust a centralized third-party (such as an RPC provider) to provide Ethereum data
+
+The Portal Network introduces a third option for Ethereum users: 
+- access Ethereum data in a decentralized way with minimal hardware and bandwidth requirements.
+
+The Portal Network is a peer-to-peer protocol that runs parallel to Ethereum. Ethereum data is **distributed across the Portal Network**, instead of being copied in every individual node. 
+This allows users to access Ethereum data with minimal hardware and bandwidth requirements and almost instant syncing.
+
+
+## How does Portal work?
+
+The Portal Network is actually several peer-to-peer networks (state, beacon, history, transaction gossip and canonical transaction index networks) that can be accessed using a Portal Network [client](../clients). 
```

Comment:
```suggestion
The Portal Network is actually several peer-to-peer networks (state, beacon, and history networks) that can be accessed using a Portal Network [client](../clients). 
```

Still probable that we'll implement the other two networks (tx gossip, and canonical tx indices) but I think I like leaving them out since I've begun viewing state/history/beacon as MVP